source/release-notes.txt

```diff
@@ -10,6 +10,20 @@ Release Notes
    :depth: 1
    :class: twocols
 
+1.4.4 Changelog
+---------------
+
+*Released January 29, 2024*
+
+Improvements:
+
+- Improved synthetic foreign keys to be preserved after schema refresh.
+- Fix for fully-offline deployments in airgapped environments. 
+- Fix for certain combinations of mappings, specifically two different
+  mappings at the same level of embedding. 
+- Various minor fixes and improvements to Sybase database support, CDC
+  stage metrics, verification stage, database exports and logging.
```

Comment:
[nit]

Consider adding a comma after "database exports" like: "...CDC stage metrics, verification stage, database exports, and logging."

Reply:
Done!